React/wbooks/src/redux/bookDetail/reducer.js

```diff
@@ -0,0 +1,37 @@
+import { fetchActions } from './actions';
+
+const initialState = {
+  items: [],
```

Comment:
should this be an array? I think the request to fetch one book return only an object. If that's the case, initialize it with null